loki/config.yaml

```
@@ -0,0 +1,65 @@
+  max_look_back_period: 0s
+
+table_manager:
+  retention_deletes_enabled: false
+  retention_period: 0s
+
+query_range:
+  results_cache:
+    cache:
+      embedded_cache:
+        enabled: true
+        max_size_mb: 100
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)